src/plugins/Polls/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { createTypedMessageMetadataReader, createRenderWithMetadata } from '../../protocols/typed-message/metadata'
+import type { PollMetaData } from './types'
+
+export const PollMetadataReader = createTypedMessageMetadataReader<PollMetaData>('com.maskbook.poll:1', {})
```

Comment:
If you don't have a JSON schema(2nd parameter), don't provide it, or the metadata might be dropped